app/uk/gov/hmrc/ccn2connectivitytester/repositories/SoapMessageStatusRepository.scala

```diff
@@ -74,18 +74,18 @@ class SoapMessageStatusRepository @Inject() (mongoComponent: MongoComponent, app
     collection.withReadPreference(primaryPreferred())
       .findOneAndUpdate(
         filter = equal("messageId", Codecs.toBson(messageId)),
-        update = set("status", Codecs.toBson(newStatus.entryName)),
+        update = set("status", Codecs.toBson(newStatus.toString())),
         options = FindOneAndUpdateOptions().upsert(true).returnDocument(ReturnDocument.AFTER)
       ).toFutureOption()
   }
 
   def retrieveMessagesMissingConfirmation: Source[SoapMessageStatus, NotUsed] = {
     MongoSource(collection.withReadPreference(primaryPreferred())
-      .find(filter = and(equal("status", SENT.entryName), and(lte("createDateTime", now().minus(appConfig.confirmationWaitDuration))))))
+      .find(filter = and(equal("status", SENT.toString()), and(lte("createDateTime", now().minus(appConfig.confirmationWaitDuration))))))
```

Comment:
Ditto this could be wrapped in Codecs.Bson if the fmtr was in scope.  Starting to wonder if Codecs.Bson crept in to too many places (not here)